{
    "config": {
        "abort": {
            "already_configured": "Az eszk\u00f6z m\u00e1r konfigur\u00e1lva van",
            "connection_error_during_import": "Csatlakoz\u00e1si hiba t\u00f6rt\u00e9nt a yaml konfigur\u00e1ci\u00f3 import\u00e1l\u00e1sa sor\u00e1n",
            "reauth_successful": "Az \u00fajrahiteles\u00edt\u00e9s sikeres volt."
        },
        "error": {
            "connection_error": "Sikertelen csatlakoz\u00e1s",
            "invalid_auth": "\u00c9rv\u00e9nytelen hiteles\u00edt\u00e9s"
        },
        "flow_title": "Nextcloud",
        "step": {
            "reauth_confirm": {
                "data": {
                    "password": "Jelsz\u00f3",
                    "username": "Felhaszn\u00e1l\u00f3n\u00e9v"
                },
                "description": "Friss\u00edtse a bejelentkez\u00e9si adatait {url} sz\u00e1m\u00e1ra."
            },
            "user": {
                "data": {
                    "password": "Jelsz\u00f3",
                    "url": "URL",
                    "username": "Felhaszn\u00e1l\u00f3n\u00e9v",
                    "verify_ssl": "SSL-tan\u00fas\u00edtv\u00e1ny ellen\u0151rz\u00e9se"
                },
                "description": "Adja meg Nextcloud adatait."
            }
        }
    },
    "issues": {
        "deprecated_yaml": {
            "description": "A Netxcloud YAML haszn\u00e1lat\u00e1val val\u00f3 konfigur\u00e1l\u00e1sa elavult.\n\nA megl\u00e9v\u0151 YAML-konfigur\u00e1ci\u00f3 automatikusan import\u00e1lva lett a felhaszn\u00e1l\u00f3i fel\u00fcletre.\n\nT\u00e1vol\u00edtsa el a \"nextcloud\" YAML-konfigur\u00e1ci\u00f3t a configuration.yaml f\u00e1jlb\u00f3l, \u00e9s ind\u00edtsa \u00fajra a Home Assistant alkalmaz\u00e1st a probl\u00e9ma megold\u00e1s\u00e1hoz.",
            "title": "A Netxcloud YAML konfigur\u00e1ci\u00f3ja elavult"
        }
    }
}